### ISSUE TITLE
Track the leaderboard position that a run would have been

### ISSUE THREAD

**Issue by TNTreee** [Vanawy/tm-run-history#9]
It can be useful to see what rank a run would have been, even though it was slower than your PB.
thus,
Run History should display rank as well as delta.

**Comment by TNTreee** [comment]
If this is added you would need to make it toggleable, and also probably make delta time togglable

**Comment by Vanawy** [comment]
> If this is added you would need to make it toggleable, and also probably make delta time togglable

Yeah, if i will add it. I will make table layout customizable so you can switch on columns you want :+1:

**Comment by Vanawy** [comment]
From Discord:

_XertroV — Today at 4:11 PM_

> you need to call the surround API from an account that hasn't got a PB on that track. I have a route on map-monitor that will do this, but it's expensive since each call requires an API call back to nadeo. The route is https://map-monitor.xk.io/map/<str:map_uid>/<int:score>/refresh

_vanawy — Today at 4:19 PM_

> Sounds slow 😔 I’m not sure if it’s worth it. Thanks for the insight

_vanawy — Today at 4:24 PM_

> Is this call used to determine place of personal best?:Nerdge:

_XertroV — Today at 4:28 PM_

> i think so, it returns your PB rank if you request a score worst than your PB (from memory, not 100% sure on that, anyway it's important that the requesting account does not have a PB if you want an accurate result, which makes the players account not very useful)
